components/quick_switch_modal/quick_switch_modal.test.tsx

```diff
@@ -35,7 +35,7 @@ describe('components/QuickSwitchModal', () => {
         it('should do nothing if nothing selected', () => {
             const props = {...baseProps};
 
-            const wrapper = shallow(
+            const wrapper: ShallowWrapper<any, any, QuickSwitchModal> = shallow(
```

Comment:
You can actually save some typing here by adding the parameter to `shallow` instead of the `ShallowWrapper`
```suggestion
            const wrapper = shallow<QuickSwitchModal>(
```
The same thing applies for all of these